packages/api/src/schema/api-schema.yaml

```
@@ -3578,7 +3856,11 @@ paths:
                 $ref: "#/components/schemas/error"
   /data/views/query:
     get:
+      operationId: getViewershipMetrics
+      x-speakeasy-name-override: getViewership
       summary: Query viewership metrics
+      tags:
+        - metrics
```

What if it was more like this instead?
```suggestion
      operationId: getViewershipMetrics
      x-speakeasy-name-override: getMetrics
      summary: Query viewership metrics
      tags:
        - views
```

---

Love the idea! I will open a separate PR to update the tags :)